### PR TITLE
[Feature] - Added Alerts component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named `alerts` under which markup has been written in `alerts.html` and the stylesheet file is written in `alerts.css`. The stylesheet file `alerts.css` only consists of the styles related to alerts i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main `styles.css` file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds a `Alerts` component of different types.

### CODE REVIEW

File: alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  text-align: center;
+  width: 15rem;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  font-weight: lighter;
+  margin: 1rem;
+}
+
+.alert-primary {
+  background-color: var(--sky-blue);
```
Comment (bharati-21) [inline alerts/alerts.css:12]: If these colors are common to all the components, please add them to a common stylesheet. If not, looks good!

File: alerts/alerts.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (bharati-21) [inline alerts/alerts.html:8]: You can add the meta author and description tag to your file.

**Review by bharati-21 — APPROVED** [review]
I have left a few suggestions. Apart from that, everything looks good to me! 

Great UI. Good choice of colors and fonts!

**Review by sonishreyas — COMMENTED** [review]
Nothing more to add in, rest all look good to me

File: alerts/alerts.css (hunk 1 above)
Comment (parthpandyappp) [reply alerts/alerts.css:12]: Actually, these colors are coming from the main stylesheet `styles.css`. Those are defined in the root styling section itself. I mean all the colors or anything which is shared among various markups is defined into a common main stylesheet i.e. `styles.css` in the root directory.

**Comment by parthpandyappp** [comment]
> I have left a few suggestions. Apart from that, everything looks good to me!
> 
> Great UI. Good choice of colors and fonts!

@bharati-21 Thank you for the appreciation. I really look forward to your opinion, glad to have them all.
Also, thanks for reviewing my PR. :smile:

**Comment by parthpandyappp** [comment]
> Nothing more to add in, rest all look good to me

@sonishreyas Thanks Shreyas for reviewing my PR :smile:

File: alerts/alerts.html (hunk 2 above)
Comment (parthpandyappp) [reply alerts/alerts.html:8]: Okay, added.

**Pull request merged**